lib/classes/popover.js

```diff
@@ -70,7 +70,7 @@ class PopOver extends ToolTip {
     getTitle() {
         let title = this.$config.title || '';
         if (typeof title === 'function') {
-            title = title.call(this.$element).toString().trim();
+            title = title(this.$element).toString().trim();
```

Comment:
this pass element as argument to function... while previous variant executed function with context of element. R U sure that this will work?

Reply:
I changed this to allow the function that produces content to maintain its own `this` context (i.e. the app context, so functions can reference app methods and data using `this`)

Reply:
And I need to remove the toString stuff from here